examples/advanced/vertical_federated_learning/cifar10-splitnn/cifar10_split_learning.ipynb

```diff
@@ -129,16 +148,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "id": "21a6b36f-649f-4e19-ba0a-5dd71dfda5dc",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
```

Comment:
do we need to keep the outputs here?

Reply:
Yes, I think it's useful to keep this for illustrative purposes when just reading through the notebook. It shows the accuracy of PSI.
```
gt_overlap [11841 19602 45519 ... 47278 37020  2217] n=10000
psi_overlap_1 [ 4481. 45431. 46253. ... 34846.   179.  7277.] n=10000
psi_overlap_2 [38639. 10733. 31911. ... 12172. 46167.   865.] n=10000
Found 100.0% of the overlapping sample ids for site-1.
Found 100.0% of the overlapping sample ids for site-2.
```

Reply:
makes sense